build-ps/debian/percona-server-server.postinst

```diff
@@ -182,6 +201,29 @@ EOF
 			echo "Please use etc/mysql/mysql.conf.d for any custom configuration settings"
 		fi
 		update-alternatives --force --install /etc/mysql/my.cnf my.cnf "/etc/mysql/mysql.cnf" 300
+		if [ -f "/etc/apparmor.d/usr.sbin.mysqld" ]; then
```

Comment:
This part seems to be repeated twice. Maybe it would be possible to extract it to a common function?